src/core/border_router/routing_manager.hpp

```diff
@@ -282,6 +282,11 @@ class RoutingManager : public InstanceLocator
     ExternalPrefix mDiscoveredPrefixes[kMaxDiscoveredPrefixNum];
     uint8_t        mDiscoveredPrefixNum;
 
+    // The RA header and parameters for the infra interface.
+    // This value is initialized with `RouterAdvMessage::SetToDefault`
+    // and updated with RA messages initiated from infra interface.
```

Comment:
What happens when the infrastructure interface stops advertising?

Reply:
Per https://tools.ietf.org/html/rfc2461#section-6.2.5 a Router SHOULD send RAs whith zero Router Lifetime when it ceases advertising. We can reset `mInfraIfRouterAdvMessage` to the default value in case a RA with zero Router Lifetime is received.  I included this in the latest push.

Reply:
But there are still cases that the RA daemon may crash before sending RAs with zero Router Lifetime. I am working on the `STALE_RA_TIME` and `Vicarious Router Discovery` described in https://tools.ietf.org/html/draft-lemon-stub-networks-02#section-3.1.1 and with this we can find out that the RA daemon is no longer alive after `STALE_RA_TIME` which is typically smaller than the Router Lifetime.